pkgs/tools/text/relpipe/default.nix

```diff
@@ -0,0 +1,192 @@
+  };
+
+  tr-cut = mkRelPipe {
+    pname = "tr-cut";
+    hash = "0ikw34irv10mgngipv1i0ha670lg695srj3mgvn23z9w1z2pjsy7";
+    deps = [ lib-cli lib-common lib-reader lib-writer ];
+  };
+  tr-grep = mkRelPipe {
+    pname = "tr-grep";
+    hash = "0cndlwmq34b9yh4i1wpf1x8yj6mqqjq7yxh01jp6p5mm10cbyymi";
+    deps = [ lib-cli lib-common lib-reader lib-writer ];
+  };
+  tr-sed = mkRelPipe {
+    pname = "tr-sed";
+    hash = "1wbw5hpich788gmsy8xfpjrmi4clx1km6wrf4w0qh72vl7w64b40";
+    deps = [ lib-cli lib-common lib-reader lib-writer ];
+  };
+  tr-awk = mkRelPipe {
+    pname = "tr-awk";
+    hash = "1bp5wmkikmj8fwmggkjb9ka4yii1qiwy1322z4qk31xx6ma6mz17";
+    deps = [ lib-cli lib-common lib-reader lib-writer ];
+  };
+  tr-scheme = mkRelPipe {
+    pname = "tr-scheme";
+    hash = "1s51z7r46gqa71i66yxkymh95djxgll6g26xhigq7z19xhgdnpdh";
+    deps = [ lib-cli lib-common lib-reader lib-writer guile_2_2 ];
+  };
+  tr-sql = mkRelPipe {
+    pname = "tr-sql";
+    hash = "0pivc1q9hvvai52d5xzfqlvbv1s4m3y7p6bj3pyb7kn38ga66fa3";
+    deps = [ lib-cli lib-common lib-reader lib-writer unixODBC ];
+  };
+  tr-python = mkRelPipe {
+    pname = "tr-python";
+    hash = "1yq1v3i2nks45rlkld57h03ii6szfa8qim5pvh2f16ziql3xmyx4";
+    deps = [ lib-cli lib-common lib-reader lib-writer python36 ];
+  };
+  tr-validator = mkRelPipe {
+    pname = "tr-validator";
+    hash = "0ac6c907aqfnmm3q3b3lcspfd430gjhzajgxay0syli5k4s1zkcc";
+    deps = [ lib-cli lib-common lib-reader lib-writer ];
+  };
+
+  out-nullbyte = mkRelPipe {
+    pname = "out-nullbyte";
+    hash = "1nq2icfs0rxzp1vdhl9nimg5cj8w478nzkyq2d9mnq79gkrdag04";
+    deps = [ lib-cli lib-common lib-reader ];
+  };
+  out-ods = mkRelPipe {
+    pname = "out-ods";
+    hash = "0i12w8cbz485v9hymhn8vxxj24cwpa0r59y2hl6m0v7mn50gnrfd";
+    deps = [ lib-cli lib-common lib-reader lib-xmlwriter ];
+  };
+  out-tabular = mkRelPipe {
+    pname = "out-tabular";
+    hash = "1idvndyvq6i5wjii14pbphn79fpc6hl28qgksxyda77l2qfynnb7";
+    deps = [ lib-cli lib-common lib-reader ];
+  };
+  out-xml = mkRelPipe {
+    pname = "out-xml";
+    hash = "0rc0xnc4g3wpqhdc6lfb1sagb1vl4yzvh2hd6q96jk45y037qs0c";
+    deps = [ lib-cli lib-common lib-reader lib-xmlwriter ];
+  };
+  out-csv = mkRelPipe {
+    pname = "out-csv";
+    hash = "1aci7y5vfi6q6s3mqw6yzavvxjhrsinsv9qixqdgbczp3lcl3bj7";
+    deps = [ lib-cli lib-common lib-reader ];
+  };
+  out-asn1 = mkRelPipe {
+    pname = "out-asn1";
+    hash = "01iazvr8qq0nbnap56zs18c3wpxaldkcffxfsp0ii6khv90yy8ka";
+    deps = [ lib-cli lib-common lib-reader ];
+  };
+  out-recfile = mkRelPipe {
+    pname = "out-recfile";
+    hash = "1d9d25w0p8vlsa5n5x3k8kb9pms06r1vwanwxx45k09l2msvw7c9";
+    deps = [ lib-cli lib-common lib-reader ];
+  };
+  out-jack = mkRelPipe {
+    pname = "out-jack";
+    hash = "16q9ck2qwyy8f7yqcaab1dicwav325cmajqah9q89xrlkjx3mq2j";
+    deps = [ lib-cli lib-common lib-reader libjack2 ];
+  };
+in symlinkJoin {
+  name = "relpipe";
+  paths = [
```

Comment:
It would be nice if it was possible to disable some of these packages, to spare some closure size for those who wish so.

Reply:
Sure. Or we could split them into an attrset. The upstream intent is that a metapackage will have all of the utilities, but there's nothing wrong with exposing them ala carte.